### ISSUE TITLE
Update Policy Modification

### ISSUE THREAD

**Issue by 597358816** [hiyouga/EasyR1#220]
In the function of update_policy when computing pg_loss, we use VF.masked.mean.
`final_pg_loss = VF.masked_mean(final_pg_loss, response_mask)`

As the author mentioned, this is the version of DAPO：

![Image](https://github.com/user-attachments/assets/4ecc7a4c-612f-4c58-8bd2-5e1950cecc73)

But current community experiments have shown that the summation order of Dr. GRPO may have a slight advantage:(remove the red part)

![Image](https://github.com/user-attachments/assets/6370ea11-36b6-4ab9-af47-bfb5dc5e6c1d)

May I ask if the author can provide a modification method for me to test? I will provide timely feedback on the test results (which version is better)

**Comment by hiyouga** [comment]
please modify this by yourself
